docs/docs/command-line-interface.mdx

```diff
@@ -23,6 +23,7 @@ abstract: The command line interface (CLI) gives you easy-to-remember commands f
 |`rasa data convert`     |Converts training data between different formats.                                                                                         |
 |`rasa data validate`    |Checks the domain, NLU and conversation data for inconsistencies.                                                                                             |
 |`rasa export`           |Exports conversations from a tracker store to an event broker.                                                                            |
+|`rasa evaluate markers` |Applies markers to existing tracker store.                                                                                                |
```

Comment:
> Applies markers to existing tracker store.

I think this is a bit too vague (and I'm slightly confused that the command is `rasa evaluate markers`, but the explanation says `Applies`, from the command I'd expect that it gives me some sort of performance measure?)

Reply:
There was a discussion [here](https://rasa-hq.slack.com/archives/C01HHMR4X8S/p1634044314193800) about what the call the command. To summarise, we expect to release a number of evaluation tools, the first of which is Markers.

I'll rephrase this 👍